environment.yml

```diff
@@ -0,0 +1,119 @@
+name: shared-latent-space
```

Comment:
how did you get all of these dependencies?? Generally, I try to keep my `environment.yml` files simple. Keeping it simple makes it quite easy for someone to view all of the package requirements of a project right away. Plus, it is easier to maintain.

Also, are you using python 2.7? Many of these are specific to python 2.7

Reply:
I just loaded the dependencies when I created the conda environment as specified here: https://conda.io/docs/user-guide/tasks/manage-environments.html#id1

I think it's because each of them have their own dependencies?

I am using Python 2.7, I have that as one of the lines in the `enviroment.yml`

Reply:
How did you generate these dependencies? Did you use a command like `conda env list` or something related to that?

Reply:
I used `conda env export > environment.yml`

Reply:
Ah I see the issue. @gwaygenomics is saying don't use conda export command because it adds a lot extra information that one may not need (e.g. py27hashes). In this case I think the best way to move forward is to only have packages that are specific towards this project. Lets chat about this, so we can sort it out.